assets/js/pages/Login/Login.jsx

```diff
@@ -55,59 +68,91 @@ export default function Login() {
-                  className={classNames(
-                    'appearance-none block w-full px-3 py-2 border border-gray-300 rounded-md shadow-sm placeholder-gray-400 focus:outline-none focus:ring-jungle-green-500 focus:border-jungle-green-500 sm:text-sm',
-                    { 'border-red-300': isUnauthorized }
-                  )}
-                />
-              </div>
-            </div>
+            {!totpCodeRequested ? (
+              <>
+                <div>
+                  <label
+                    htmlFor="username"
+                    className="block text-sm font-medium text-gray-700"
+                  >
+                    Username
+                  </label>
+                  <div className="mt-1">
+                    <input
+                      id="username"
+                      type="text"
+                      data-testid="login-username"
+                      disabled={authInProgress}
+                      value={username}
+                      onChange={(e) => setUsername(e.target.value)}
+                      name="username"
+                      autoComplete="username"
+                      required
+                      className={classNames(
+                        'appearance-none block w-full px-3 py-2 border border-gray-300 rounded-md shadow-sm placeholder-gray-400 focus:outline-none focus:ring-jungle-green-500 focus:border-jungle-green-500 sm:text-sm',
+                        { 'border-red-300': isUnauthorized }
+                      )}
+                    />
+                  </div>
+                </div>
 
-            <div>
-              <label
-                htmlFor="password"
-                className="block text-sm font-medium text-gray-700"
-              >
-                Password
-              </label>
-              <div className="mt-1">
-                <input
-                  id="password"
-                  type="password"
-                  data-testid="login-password"
-                  disabled={authInProgress}
-                  value={password}
-                  onChange={(e) => setPassword(e.target.value)}
-                  autoComplete="current-password"
-                  required
-                  className={classNames(
-                    'appearance-none block w-full px-3 py-2 border border-gray-300 rounded-md shadow-sm placeholder-gray-400 focus:outline-none focus:ring-jungle-green-500 focus:border-jungle-green-500 sm:text-sm',
-                    {
-                      'border-red-300': isUnauthorized,
-                      'disabled:opacity-50': authInProgress,
-                    }
-                  )}
-                />
+                <div>
+                  <label
+                    htmlFor="password"
+                    className="block text-sm font-medium text-gray-700"
+                  >
+                    Password
+                  </label>
+                  <div className="mt-1">
+                    <input
```

Comment:
Could we use the design system Input component, instead of the html one?

If this is a tech debt from the past was because originally the login page was not made with react, I think could be a good opportunity to change, or at least add the otp one as react component and then have a dedicated tech debt for the others

Reply:
Alright!
Yes, it was a tech debt, and I didn't want to change in this PR, but the change is minimal, so I will finally do it.

Reply:
@CDimonaco Done!
If you notice that I use `Input` component for the password instead of `Password`, this is because if I use this 2nd, the classes are not inherited properly...
This is because the `Password` uses the `suffix` prop to add the "show password icon".
When you do this, the classes are not passed to the correct component, as rc-input creates an intermediate `span` element that gets the classes. A mess...
It is a known limitation.
As far as I researched, there might be some workarounds, but I don't want to start looking at them now.
As we don't have the icon now, not adding it doesn't look like an urgent thing

Reply:
Let's do that in another pr, it's fine, we just need to remember this debt